### DISCUSSION TITLE
Documentation update for spaceship - AppStoreConnect

### DISCUSSION THREAD

**Discussion by dorgalam** [fastlane/fastlane#18580] [General]
Hi folks!

I'm trying to use spaceship as a library in my ruby code and I've found that the docs aren't up to date.

Specifically, these docs: https://github.com/fastlane/fastlane/blob/master/spaceship/docs/AppStoreConnect.md#appversions
Ask you to use `Spaceship::ConnectAPI::App` and then `get_live_app_store_version`, but then refer to changes as if the returned value is the class `AppVersion` rather than `AppStoreVersion`.

It is my understanding that `AppVersion` is that some old api uses, and `AppStoreVersion` is the updated and recommended one.

I would love some clarity in case I'm wrong.
Thanks!

**Comment by lucgrabowski** [comment]
I've opened a PR with docs update: https://github.com/fastlane/fastlane/pull/18969

**Reply by lucgrabowski** [reply]
Update for AppStoreConnect docs was merged lately @dorgalam.
